src/JustSaying/AwsTools/MessageHandling/SnsTopicBase.cs

```diff
@@ -136,5 +137,10 @@ private static MessageAttributeValue BuildMessageAttributeValue(Messaging.Messag
                 DataType = value.DataType
             };
         }
+
+        public virtual InterrogationResult Interrogate()
+        {
+            return new InterrogationResult(new {});
```

Comment:
I wonder for this if we could make an internal static `InterrogationResult.Empty` to use for this?

Reply:
Sure thing, that sounds like a good idea 👍

Reply:
Done